UI/js/common.js

```diff
@@ -14,11 +14,33 @@ const errors = Object.freeze({
     2002: 'Invalid username or password',
     2003: 'Invalid username or password',
     2004: 'Invalid username or password',
-    2005: 'Account is not active'
+    2005: 'Account is not active',
+    2006: 'Session timed out',
+    2007: 'failed to update user, missing information'
```

Comment:
upper case in the beginning of the UI errors please `failed` -> `Failed`

Reply:
nice catch, I will do it for both the common files